v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.monster.etl.v2f
+
+/**
+  * constants for v2f that allow for transforms of the converted tsv to json
+  */
+sealed trait V2FConstants {
+
+  /**
+    * the name of the tsv table that is being converted to Json and transformed
+    */
+  def tableName: String
+
+  /**
+    * relative path to the tsv that will be converted to Json and transformed
+    */
+  def tsvPath: String
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
+    */
+  def fieldsToConvertToJsonDouble: List[String]
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
+    */
+  def fieldsToConvertToJsonInt: List[String]
+
+  /**
+    * the names of json fields that should be converted from a json string to a json boolean
+    */
+  def fieldsToConvertToJsonBoolean: List[String]
+
+  /**
+    * the names json fields that should be converted from from a json string to a json array
+    * the keys of the map are delimeter's and the values of map are names of json fields for the give delimeter(key)
+    */
+  def fieldsToConvertToJsonArray: Map[String, List[String]]
+
+  /**
+    * the json fields of a json array that should be converted from a json string to a json double
+    */
+  def fieldsToConvertFromJsonArrayStringToDouble: List[String]
+}
+
+case object FrequencyAnalysis extends V2FConstants {
+  override def tableName = "Frequency Analysis"
+
+  override def tsvPath: String = "frequencyanalysis/*/*"
```

Comment:
We control the layout of the raw TSVs in cloud storage (via Transporter), so we could make this a little more regular. I'd also lock it down to only CSV files:
```suggestion
  override def tsvPath: String = "frequency-analysis/*/*.csv"
```